onnxruntime/python/_pybind_state.py

```diff
@@ -5,9 +5,23 @@
 
 import os
 import platform
+import sys
 import warnings
 import onnxruntime.capi._ld_preload  # noqa: F401
 
+# Python 3.8 (and later) on Windows doesn't search system PATH when loading DLLs,
+# so CUDA location needs to be specified explicitly.
+if platform.system() == "Windows" and sys.version_info >= (3, 8):
+    CUDA_VERSION = "10.2"
```

Comment:
What was the reason to use the specific version? Not path from variable `CUDA_PATH` ?